onyo/commands/tests/test_mkdir.py

```diff
@@ -132,17 +132,17 @@ def test_mkdir_quiet_flag(repo: OnyoRepo) -> None:
 
 @pytest.mark.repo_dirs(*directories)
 @pytest.mark.parametrize('directory', directories)
-def test_error_dir_exists(repo: OnyoRepo, directory: str) -> None:
+def test_dir_exists(repo: OnyoRepo, directory: str) -> None:
     """
     Test the correct error behavior when `onyo mkdir <path>` is called on an
```

Comment:
Nit picky, but is it still "error behavior" or rather just behavior? :)

Reply:
```suggestion
    Test the correct behavior when `onyo mkdir <path>` is called on an
```